sdk/maps/azure-maps-search/azure/maps/search/aio/_patch.py

```diff
@@ -26,26 +45,20 @@ def _authentication_policy(credential):
         )
     return authentication_policy
 
-class AsyncMapsSearchClientBase:
+
+# pylint: disable=C4748
+class MapsSearchClient(MapsSearchClientGenerated):
+
     def __init__(
         self,
         credential: Union[AzureKeyCredential, AsyncTokenCredential],
         **kwargs: Any
```

Comment:
Same as for the sync client:
```suggestion
        *,
        endpoint: Optional[str] = None,
        client_id: Optional[str] = None,
        **kwargs: Any
```